### PR TITLE
[DOC]Add modelscope example

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

feature or bug fix or documentation or others
API changed or not
not
## Description
Add modelscope example

### CODE REVIEW

File: examples/modelscope/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# ModelScope with ITREX
+
+Intel extension for transformers(ITREX) support almost all the LLMs in Pytorch format from ModelScope such as phi,Qwen,ChatGLM,Baichuan,gemma,etc.
+
+## Usage Example
+
+ITREX provides a script that demonstrates the use of modelscope. Run it with the following command:
+```bash
+numactl -m 0 -C 0-55 python run_modelscope_example.py --model_path=qwen/Qwen-7B --prompt=你好
```
Comment (zhentaoyu) [inline examples/modelscope/README.md:9]: numactl -l -C xx-xx? if user across sockets? 
Add a note explaining why adding numactl is necessary (to improve performance and teach them how to bind core_id).

**Review by zhentaoyu — APPROVED** [review]

File: examples/modelscope/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# ModelScope with ITREX
+
+Intel extension for transformers(ITREX) support almost all the LLMs in Pytorch format from ModelScope such as phi, Qwen, ChatGLM, Baichuan, gemma, etc.
+
+## Usage Example
+
+ITREX provides a script that demonstrates the use of modelscope. Run it with the following command:
+```bash
+numactl -m 0 -C 0-55 python run_modelscope_example.py --model_path=qwen/Qwen-7B --prompt=你好
```
Comment (a32543254) [inline examples/modelscope/README.md:9]: numactl -m 0 -C 0-55 python run_modelscope_example.py --model_path=qwen/Qwen-7B --prompt=你好
change to 
OMP_NUM_THREADS=<physical cores num> numactl -m <node N> -C <cpu list>  python run_modelscope_example.py \
    --model <MODEL_NAME_OR_PATH> \
    --prompt=你好

File: examples/modelscope/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+```
+
+## Supported and Validated Models
+We have validated the majority of existing models using modelscope==1.13.1:
```
Comment (a32543254) [inline examples/modelscope/README.md:13]: please add the requirment.txt

**Review by a32543254 — APPROVED** [review]
LGTM

File: examples/modelscope/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# ModelScope with ITREX
+
+Intel extension for transformers(ITREX) support almost all the LLMs in Pytorch format from ModelScope such as phi, Qwen, ChatGLM, Baichuan, gemma, etc.
```
Comment (a32543254) [inline examples/modelscope/README.md:3]: Intel extension for transformers
Intel® Extension for Transformers

**Pull request merged**